### PR TITLE
Make datetime format configurable

### PR DESCRIPTION
**Pull request by FaulTTY**

### CODE REVIEW

File: src/Config.h [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Config
     std::string batteryFolder = ""; // this can be BAT0, BAT1, etc. Usually in /sys/class/power_supply
     std::vector<std::string> workspaceSymbols = std::vector<std::string>(9, "");
     std::string defaultWorkspaceSymbol = "";
+    std::string DateTimeStyle = "%a %D - %H:%M:%S %Z"; // A sane default
```
Comment (scorpion-26) [inline src/Config.h:18]: Lowercase variable please (dateTimeStyle)

File: src/System.cpp [hunk 2]
```diff
@@ -604,7 +604,9 @@ namespace System
         time_t stdTime = time(NULL);
         tm* localTime = localtime(&stdTime);
         std::stringstream str;
-        str << std::put_time(localTime, "%a %D - %H:%M:%S %Z");
+        std::string dateTimeStyle = Config::Get().DateTimeStyle;
+        const char* charPtr_dateTimeStyle = dateTimeStyle.c_str();
+        str << std::put_time(localTime, charPtr_dateTimeStyle);
```
Comment (scorpion-26) [inline src/System.cpp:609]: Can you merge these three lines together ( -> Config::Get().dateTimeStyle.c_str())

**Comment by scorpion-26** [comment]
Thanks for the PR! I'll merge once everything is resolved.

**Comment by FaulTTY** [comment]
Sure thing, no problem. Thanks for the project!

**Comment by scorpion-26** [comment]
Nice, LGTM.

**Pull request merged**